### DISCUSSION TITLE
macOS transparency incorrect after switching back from full screen

### DISCUSSION THREAD

**Discussion by docbobo** [ghostty-org/ghostty#3600] [Q&A]
I am using the following transparency settings on macos:

```
background-opacity = 0.8
background-blur-radius = 32
```
When switching to native full screen, transparency gets disabled (due to #2840/#2842?) However, when returning back from full screen, transparency is not restored and the window stays opaque.

**Comment by mitchellh** [comment]
This is a dupe. I don't have the original issue number right now, but this was reported in the last 24 hours.

**Reply by docbobo** [reply]
Looked through both issues and discussions and could not find anything. Maybe it was reported on Discord? Anyway, if it's on your radar, it's on your radar...

**Reply by docbobo** [reply]
@mitchellh this seems to be fixed for most of the window except for the titlebar. When returning from fullscreen on macOS, the titlebar becomes opaque.

**Reply by docbobo** [reply]
Small correction: this happens with the fake fullscreen mode, not the native one. The latter seems to work.
